bluefog/torch/mpi_ops.py

```diff
@@ -1007,10 +1007,17 @@ def win_wait(handle: int) -> bool:
     return True
 
 
-def get_win_version(name: str):
-    """Get the version of the window in each node."""
+def get_win_version(name: str) -> Dict[int, int]:
+    """Get the version of the window in each node and return a dictionary between the rank and the version.
```

Comment:
Too long.

Get a dictionary maps from rank to the version of stored win value.